reps/2023-04-28-remove-algorithms-from-rllib.md

```diff
@@ -0,0 +1,193 @@
+## Summary
+
+We'd like to remove aprox ~20 of the 30 algorithms from RLlib. We are considering doing this because 
+
+1. It would reduce the maintenance burden of RLlib.
```

Comment:
the most important consideration is to lower the barrier for the community to contribute all kinda of algorithms?
since they can be pinned to different versions of RLlib now.

Reply:
Yes we should highlight this new value proposition and use it as a strong reason.